examples/explainable_ai/steps/explainer.py

```diff
@@ -0,0 +1,84 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+from torch.utils.data import DataLoader
+from torch import nn
+import torch
+from typing import List, Dict
+
+import numpy as np
+import matplotlib
+from foxai.context_manager import FoXaiExplainer, ExplainerWithParams, CVClassificationExplainers
+from foxai.visualizer import mean_channels_visualization
+
+from zenml import step
+from zenml.steps import Output
+
+
+@step
+def explain(model: nn.Module, test_dataloader: DataLoader) -> Output(
+        explanation=List,
+        figure_list=List,
+):
```

Comment:
Can this maybe return an object that can be visualized in the dashboard? That would be super cool. See docs here: https://docs.zenml.io/user-guide/advanced-guide/visualize-artifacts#creating-custom-visualizations

Reply:
I was able to generate visualizations for figures using `mpld3` library to convert `matplotlib.figure.Figure`: https://github.com/zenml-io/zenml/pull/1663/commits/332b4eabfe65784b0017cae22750db1d08a1de5f.

Reply:
Isnt that super cool!? Nice!

Reply:
I've added an example with the shap library: https://github.com/zenml-io/zenml/pull/1663/commits/5470d5c95246167afc68f5c8ea329c0df47ed175. Unfortunately visualization from shap is clipped, as it is quite large in width. There are no slides to move figure. I had to download raw HTML and display it in the browser.

In zenml:
![Screenshot from 2023-07-03 11-59-10](https://github.com/zenml-io/zenml/assets/19324675/5db75e9a-3681-4464-9332-fe9959a8fdd8)

Downloaded and visualized in browser:
![Screenshot from 2023-07-03 11-59-20](https://github.com/zenml-io/zenml/assets/19324675/52396d0d-e488-4562-8ed3-1ece56dd6b5e)

Reply:
Is it possible to add this horizontal sliding utility to the visualization page? It would make the visualization of large images easier for the user.

Reply:
@Cahllagerfeld Can you maybe take a look if its possible to do this in the dashboard easily?